todo_app/src/main/java/org/pahappa/systems/task/Task.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * 
+ */
+/**
+ * @author gilbert_yiga
+ *
+ */
+package org.pahappa.systems.task;
+
+
+import org.pahappa.systems.constants.Status;
+/**
+ * 
+ * the task class holds the attributes of the tasks
+ *
+ */
+public class Task {
```

Comment:
It is better to do documentation for the getters and setters, and then add comments to the variables.